src/test/java/com/rultor/agents/daemons/StartsDaemonITCase.java

```diff
@@ -171,11 +167,24 @@ private Talk talk() throws IOException {
                 .add("daemon").attr("id", "fedcba")
                 .add("title").set("some operation").up()
                 .add("script").set(
-                    String.format("ls -al; %s file.bin; sleep 50000", executor)
-                )
+                String.format("ls -al; %s file.bin; sleep 50000", executor)
+            )
         );
+        final Agent agent = new StartsDaemon(profile);
+        agent.execute(talk);
+        return talk;
+    }
+
+    /**
+     * Creates a mock Profile with the specified {@link XMLDocument} as
+     * profile content in XML format.
+     * @param xml The content of the profile.
+     * @return A Mock profile for testing purpose.
+     * @throws IOException If an error occurs.
```

Comment:
@essobedo dots at the end of at-clauses are unnecessary and they will soon become prohibited